src/ORM/DataObject.php

```diff
@@ -940,6 +949,50 @@ public function i18n_plural_name()
         return _t(static::class . '.PLURALNAME', $this->plural_name());
     }
 
+    /**
+     * Get description for this class
+     * @return null|string
+     */
+    public function classDescription()
+    {
+        return static::config()->get('class_description', Config::UNINHERITED);
+    }
+
+    /**
+     * Get localised description for this class
+     * @return null|string
+     */
+    public function i18n_classDescription()
```

Comment:
```suggestion
    public function getI18nClassDescription()
```

Reply:
This is mixing snake_case AND camelCase :-)

We shouldn't be adding new API that's not following modern PHP coding standards

Reply:
I don't want to change these names. I'm not adding new API, I'm just moving it up the class hierarchy.

`getI18nClassDescription` is especially bad since it's then not going to pop up with I go `$this->i18n` to find the i18n methods like `i18n_pluralise()`. It should explicitly and intentionally start with `i18n` - or we should rename the other i18n methods which is not in scope for this PR.

Reply:
OK